config/routes.rb

```diff
@@ -0,0 +1,24 @@
+Rails.application.routes.draw do
+  devise_for :users, :controllers => { registrations: 'users/registrations' }
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+# put '/recipes/:id', to: 'recipes#recipe_update'
+  # Defines the root path route ("/")
+  # root "articles#index"
+
+  root "foods#index"
+  # resources :users
+  # resources :foods
+  # resources :recipes
+  # resources :recipe_foods
```

Comment:
- [ ] In order to keep our codebase clean and professional, kindly remove all commented lines of code.